tests/pages/firefox/all.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class FirefoxAllPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/all'
+
+    _search_input_locator = (By.ID, 'language-search-q')
+    _submit_button_locator = (By.CSS_SELECTOR, '#language-search button')
+    _build_rows_locator = (By.CSS_SELECTOR, '#builds table > tbody > tr')
+
+    @property
+    def builds(self):
```

Comment:
I would make this private, as we don't expect tests to directly access builds that may not be displayed.